Makefile

```diff
@@ -14,7 +14,7 @@
 
 EXTENSION_PREFIX            := gardener-extension
 NAME                        := os-coreos
-REGISTRY                    := eu.gcr.io/gardener-project
+REGISTRY                    := eu.gcr.io/gardener-project/gardener
```

Comment:
Can you add a release note about it?
Does it make sense also to adapt all other (OS) extensions?

Reply:
It's just about the local setup, the the official registry is already `eu.gcr.io/gardener-project/gardener`, see https://github.com/gardener/gardener-extension-os-coreos/blob/master/.ci/pipeline_definitions#L12

Reply:
Ah, ok. Thank you for the clarification.